src/Components/CreateImageWizard/CreateImageWizard.scss

```diff
@@ -97,5 +103,16 @@ div.pf-v5-c-alert.pf-m-inline.pf-m-plain.pf-m-warning {
     // flex: 1 1 0
     
     // TO DO: Remove after the fix
-    min-height: 800px;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    min-height: 800px;  /* Always go with pixels */
+}
+
+.pf-v5-c-wizard__footer {
+    position: sticky;
+    bottom: 0;
+    padding: 1rem;
+    background-color: white;  // Ensure background covers content below
```

Comment:
Was this an issue? this seem to be quite strict, PF supports changing themes, so I would avoid setting colors directly unless strictly required.

Reply:
Good point! The direct color setting isn’t strictly necessary here, and I’ve removed it to keep it theme-compatible with PatternFly.